pytopo/sweep/decorators.py

```diff
@@ -96,9 +129,9 @@ def hardsweep(ind: List[Tuple], dep: List[Tuple]) ->Callable:
     """
     Args:
         ind: List of independent parameters, defined as tuples of names and
-                units.
+                units (and optionally 'paramtype').
         dep: List of dependent parameters, defined as tuples of names and
-                units.
+                units (and optionally 'paramtype').
 
     Returns:
         A decorator which returns a sweep object, which can be directly used
```

Comment:
at least for the `hardsweep` decorator, the dictionaries that the resulting sweep object generates don't have numpy arrays. so this does not completely work as expected.

Reply:
should be fixed after some commits.